src/www/app/outline_server_repository/access_key_serialization.ts

```diff
@@ -16,35 +16,33 @@ import {makeConfig, SHADOWSOCKS_URI, SIP002_URI} from 'ShadowsocksConfig';
 
 import * as errors from '../../model/errors';
 
-import {ShadowsocksConfig} from '../config';
+import {ShadowsocksSessionConfig} from '../tunnel';
 
 // DON'T use these methods outside of this folder!
 
 // Parses an access key string into a ShadowsocksConfig object.
-export function accessKeyToShadowsocksConfig(accessKey: string): ShadowsocksConfig {
+export function accessKeyToShadowsocksSessionConfig(accessKey: string): ShadowsocksSessionConfig {
   try {
     const config = SHADOWSOCKS_URI.parse(accessKey);
     return {
       host: config.host.data,
       port: config.port.data,
       method: config.method.data,
       password: config.password.data,
-      name: config.tag.data,
     };
   } catch (error) {
     throw new errors.ServerUrlInvalid(error.message || 'failed to parse access key');
   }
 }
 
 // Enccodes a Shadowsocks proxy configuration into an access key string.
-export function shadowsocksConfigToAccessKey(config: ShadowsocksConfig): string {
+export function shadowsocksSessionConfigToAccessKey(config: ShadowsocksSessionConfig): string {
```

Comment:
This function is never used, except to convert a V0 config. See my other comment.